config/deploy.rb

```diff
@@ -1,11 +1,16 @@
 # config valid only for current version of Capistrano
-lock '3.4.0'
+lock '3.3.5'
 
-set :application, 'my_app_name'
-set :repo_url, 'git@example.com:me/my_repo.git'
+set :application, 'aggressive-epsilon'
+set :repo_url, 'git@github.com:umts/aggressive-epsilon.git'
+set :user, 'rails'
```

Comment:
strange, it asks for my password
![image](https://cloud.githubusercontent.com/assets/2114323/22756046/cf84d9c4-ee13-11e6-9ead-7620593522a3.png)